doc/htmldoc/nest_sonata/nest_sonata_guide.rst

```diff
@@ -0,0 +1,259 @@
+to be provided in HDF5 datasets with the configuration details specified in the JSON configuration file.  
+
+
+.. _sec:sonata_edges:
+
+The NEST support of SONATA edges  
+--------------------------------
+
+Analogous to nodes, edge populations are serialized in edges HDF5 files and have a single associated edge types 
+CSV file that assigns properties to all edges with a given edge type id.
+
+NEST assumes the following structure of the edges HDF5 files: 
+
+:: 
+
+    <edges_file.h5>                     Filename
+    ├─ edges                            Group - required
+    │  ├─ <population_name>             Group - required - usually only one but can be more population groups per file
+    │  │  ├─ source_node_id             Dataset {N_total_edges} - required - with attribute specifying source population name
+    │  │  ├─ edge_group_id              Dataset {N_total_edges} - required
+    │  │  ├─ edge_group_index           Dataset {N_total_edges} - required
+    │  │  ├─ target_node_id             Dataset {N_total_edges} - required - with attribute specifying target population name
+    │  │  ├─ edge_type_id               Dataset {N_total_edges} - required
+    │  │  ├─ indices                    Group - optional - currently not utilized
+    │  │  │  ├─ source_to_target        Group
+    │  │  │  │  ├─ node_id_to_range     Dataset {N_source_nodes x 2}
+    │  │  │  │  ├─ range_to_edge_id     Dataset {N_source_nodes x 2}
+    │  │  │  ├─ target_to_source        Group
+    │  │  │  │  ├─ node_id_to_range     Dataset {N_target_nodes x 2}
+    │  │  │  │  ├─ range_to_edge_id     Dataset {N_target_nodes x 2}
+    │  │  ├─ <edge_id1>                 Group - required 
+    │  │  │  ├─ delay                   Dataset {M_edges} - optional
+    │  │  │  ├─ syn_weight              Dataset {M_edges} - optional
+    │  │  │  ├─ dynamics_params         Group - currently not supported
+    │  │  ├─ <edge_id2>                 Group - optional - currently no support for more than one edge group
+    │  │  │  ├─ delay                   Dataset {K_edges} - optional
+    │  │  │  ├─ syn_weight              Dataset {K_edges} - optional
+    │  │  │  ├─ dynamics_params         Group
+
+
+Together the ``source_node_id`` and ``target_node_id`` datasets explicitly tabulate all individual connections. 
+The ``edge_type_id`` dataset attributes each edge its edge type id which is used to assign synaptic properties from the 
+edge types CSV file. 
+
+In the SONATA format, edges within a population can be organized into one or more edge groups. Synaptic properties that 
+are specified on an individual-basis are stored in these edge groups. The groups are identified by an ``edge_id`` key. 
+NEST assumes the ``edge_id``\s are contiguous numeric keys starting from zero, i.e. 0, 1, 2, ... 
+
+**Note:** NEST currently only supports one edge group per edge population. Furthermore, NEST only reads the ``delay`` 
```

Comment:
Use the Restructured Text format for notes

```
.. note::

     NEST currently  . . .

```

Reply:
Resolved by [3941e90](https://github.com/nest/nest-simulator/pull/2595/commits/3941e90f64518f2c102252ea7742b1bf5bd0bc6d).